### PR TITLE
[IT] first responses

### PR DESCRIPTION
**Pull request by iddiek**

### CODE REVIEW

**Comment by alessandroias** [comment]
I'm not absolutely sure, but I don't think answers can be templates.

**Comment by alessandroias** [comment]
And above all, being a voice assistant, shouldn't the answers be more conversational? Like "Ho Acceso [...]"?

File: responses/it/HassTurnOff.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@ language: it
...
-      default: Turned off {{ slots.name }}
-      lights_area: Turned off lights in {{ slots.name }}
-      fans_area: Turned off fans in {{ slots.name }}
-      cover: Closed {{ slots.name }}
-      cover_area: Closed {{ slots.area }}
+      default: spento {{ slots.name }}
+      lights_area: luc(e | i) spent(a | e) in {{ slots.name }}
+      fans_area: Spento <fan> in {{ slots.name }}
+      cover: chiuso {{ slots.name }}
+      cover_area: chiuso {{ slots.area }}
```
Comment (alessandroias) [inline responses/it/HassTurnOff.yaml:9]:
```suggestion
      default: Ho spento {{ slots.name }}
      lights_area: Ho spento le luci in {{ slots.area }}
      fans_area: Ho spento il ventilatore in {{ slots.area }}
      cover: Ho chiuso {{ slots.name }}
      cover_area: Ho chiuso {{ slots.area }}
```

File: responses/it/HassTurnOn.yaml [hunk 2]
```diff
@@ -2,8 +2,8 @@ language: it
...
-      default: Turned on {{ slots.name }}
-      lights_area: Turned on lights in {{ slots.area }}
-      fans_area: Turned on fans in {{ slots.area }}
-      cover: Opened {{ slots.name }}
-      cover_area: Opened {{ slots.area }}
+      default: Acceso {{ slots.name }}
+      lights_area: Luc(e|i) acces(a|e) in {{ slots.area }}
+      fans_area: <fan> acces(o|i) in {{ slots.area }}
+      cover: Aperto {{ slots.name }}
+      cover_area: Aperto {{ slots.area }}
```
Comment (alessandroias) [inline responses/it/HassTurnOn.yaml:9]:
```suggestion
      default: Ho acceso {{ slots.name }}
      lights_area: Ho acceso le luci in {{ slots.area }}
      fans_area: Ho acceso il ventilatore in {{ slots.area }}
      cover: Ho aperto{{ slots.name }}
      cover_area: Ho aperto{{ slots.area }}
```

**Review by alessandroias — COMMENTED** [review]
I would love to have something like this!

**Comment by auanasgheps** [comment]
> And above all, being a voice assistant, shouldn't the answers be more conversational? Like "Ho Acceso [...]"?

It's a good idea, but the original English version does not look that much conversational. 

example:

```
default: Turned on {{ slots.name }}
lights_area: Turned on lights in {{ slots.area }}
fans_area: Turned on fans in {{ slots.area }}
cover: Opened {{ slots.name }}
cover_area: Opened {{ slots.area }}
```

@synesthesiam could you please share some light? Do you expect the answers to be conversational like "I've turned on <stuff>" as a commercial Voice Assistant or we should stick to the original registry?

**Comment by iddiek** [comment]
For a first release - that is untestable irl at the moment - my opinion is a literal translation

**Comment by auanasgheps** [comment]
I agree. Waiting for his feedback.

File: responses/it/HassClimateSetTemperature.yaml [hunk 3]
```diff
@@ -2,4 +2,4 @@ language: it
 responses:
   intents:
     HassClimateSetTemperature:
-      default: Temperature set to {{ slots.temperature }} degrees
+      default: <temp> impostata a {{ slots.temperature }} gradi
```
[the remark this answers is not in the document]
Comment (balloob) [reply responses/it/HassClimateSetTemperature.yaml:5]: So you also cannot use `<temp>`

[the remark this answers is not in the document]
Comment (iddiek) [reply responses/it/HassClimateSetTemperature.yaml:5]: Ty

**Comment by iddiek** [comment]
Sorry @auanasgheps @xraver didn't remove you intentionally :)

File: responses/it/HassLightSet.yaml [hunk 4]
```diff
@@ -2,7 +2,7 @@ language: it
 responses:
   intents:
     HassLightSet:
-      brightness: '{{ slots.name }} brightness set to {{ slots.brightness }}'
-      brightness_area: Brightness in {{ slots.area }} set to {{ slots.brightness }}
-      color: '{{ slots.name }} color set to {{ slots.color }}'
-      color_area: Color in {{ slots.area }} set to {{ slots.color }}
+      brightness: '{{ slots.name }} luminosità impostata al {{ slots.brightness }} percento'
+      brightness_area: '[la] luminosità in {{ slots.area }} è modificata al {{ slots.brightness }} percento'
```
Comment (balloob) [inline responses/it/HassLightSet.yaml:6]: Still contains template syntax `[la]`

File: responses/it/HassLightSet.yaml [hunk 5]
```diff
@@ -2,7 +2,7 @@ language: it
...
+      color: '{{ slots.name }} colore modificato in {{ slots.color }}'
+      color_area: '[il] colore in {{ slots.area }} è modificato in {{ slots.color }}'
```
Comment (balloob) [inline responses/it/HassLightSet.yaml:8]: Still contains template syntax `[il]`

File: responses/it/HassTurnOff.yaml [hunk 6]
```diff
@@ -2,8 +2,8 @@ language: it
 responses:
   intents:
     HassTurnOff:
-      default: Turned off {{ slots.name }}
-      lights_area: Turned off lights in {{ slots.name }}
-      fans_area: Turned off fans in {{ slots.name }}
-      cover: Closed {{ slots.name }}
-      cover_area: Closed {{ slots.area }}
+      default: spento {{ slots.name }}
+      lights_area: luc(e | i) spent(a | e) in {{ slots.name }}
```
Comment (balloob) [inline responses/it/HassTurnOff.yaml:6]: Still contains template syntax `luc(e | i) spent(a | e)`

File: responses/it/HassTurnOn.yaml [hunk 7]
```diff
@@ -2,8 +2,8 @@ language: it
 responses:
   intents:
     HassTurnOn:
-      default: Turned on {{ slots.name }}
-      lights_area: Turned on lights in {{ slots.area }}
-      fans_area: Turned on fans in {{ slots.area }}
-      cover: Opened {{ slots.name }}
-      cover_area: Opened {{ slots.area }}
+      default: Acceso {{ slots.name }}
+      lights_area: Luc(e|i) acces(a|e) in {{ slots.area }}
```
Comment (balloob) [inline responses/it/HassTurnOn.yaml:6]: Same

File: responses/it/HassTurnOn.yaml [hunk 8]
```diff
@@ -2,8 +2,8 @@ language: it
...
+      fans_area: <fan> acces(o|i) in {{ slots.area }}
```
Comment (balloob) [inline responses/it/HassTurnOn.yaml:7]: Here

File: responses/it/HassTurnOff.yaml [hunk 9]
```diff
@@ -2,8 +2,8 @@ language: it
 responses:
   intents:
     HassTurnOff:
-      default: Turned off {{ slots.name }}
-      lights_area: Turned off lights in {{ slots.name }}
-      fans_area: Turned off fans in {{ slots.name }}
-      cover: Closed {{ slots.name }}
-      cover_area: Closed {{ slots.area }}
+      default: spento {{ slots.name }}
```
Comment (xraver) [inline responses/it/HassTurnOff.yaml:5]: this way to handle is different from the others.
I personally prefer to align this one to the others and make something like:
"Ho spento {{ slots.name }}"

Comment (balloob) [reply responses/it/HassTurnOff.yaml:5]: if you don't mind, I am going to merge this PR now so some responses are in place. We can tweak the content afterwards.

Comment (xraver) [reply responses/it/HassTurnOff.yaml:5]: ok

**Pull request merged**